src/detection/card/index.jsx

```diff
@@ -3,10 +3,14 @@ import React from 'react'
 import { Wrapper } from 'detection/card/wrapper'
 import { Title } from 'detection/card/title'
 import { Content } from 'detection/card/content'
-import { Buttons } from 'detection/card/action'
+import { useItems } from 'detection/card/use-items'
+
+export const Card = () => {
+  const data = useItems()
```

Comment:
Since our data comes in from a hook call, let's call this hook multiple times within each of our components, so that we don't have to pass in props.

Reply:
done

Reply:
![image](https://user-images.githubusercontent.com/23569938/129872254-6b3158a5-b3a1-4fbb-8553-1b3b2ba9163a.png)
Think you forgot to remove it from here :smile:

Reply:
done